include/uv-unix.h

```diff
@@ -72,6 +72,34 @@
 # define UV_IO_PRIVATE_PLATFORM_FIELDS /* empty */
 #endif
 
+
+// 8+6+6+6+6=32
+#define TVR_BITS  8
+#define TVN_BITS  6
+#define TVR_SIZE  (1 << TVR_BITS)
+#define TVN_SIZE  (1 << TVN_BITS)
+#define TVR_MASK  (TVR_SIZE - 1)
+#define TVN_MASK  (TVN_SIZE - 1)
+#define MAX_TVAL ((unsigned long)((1UL << (TVR_BITS + 4 * TVN_BITS)) - 1))
+
+typedef void *TQUEUE[2];
```

Comment:
Remove this and just use [QUEUE](https://github.com/libuv/libuv/blob/v1.x/src/queue.h#L21)